### ISSUE TITLE
RenPy Launcher is Completely Inaccessible to Screen Readers

### ISSUE THREAD

**Issue by PhoenixTilea** [renpy/renpy#5893]
Hi there. I am a blind developer interested in checking out the RenPy engine. That everything is done in text files means I should be able to work with sighted friends on a visual novel. However, when I start the RenPy launcher, I am unable to do anything. My screen reader simply says "RenPy Launcher", then there is no other audio feedback no matter what I do. For example, I expect to be able to use the tab key or the arrow keys to move between buttons or menu options. As far as I can tell, there is an empty window with nothing in it - which probably isn't the case.

The command-line tool seems like a step in the right direction for several project actions. Yet, there doesn't seem to be a way to actually create a new RenPy project via the CLI, which does make sense as the CLI is currently meant for CI / CD. This is a pretty big bottleneck since it leaves me with no way to create a new project.

Os: Windows 11 Professional
Screen Reader: NVDA

Either an accessible app or expanded CLI capability would be fine - I would just really like a way to actually use this engine.

**Comment by renpytom** [comment]
My impression is that screen readers are limited in that they're unable to read text rendered through OpenGL, and so you're unlikely to be able to get them working with Ren'Py. 

Ren'Py does have a self-voicing mode, which you can get to by hitting the 'v' key. It's primarily intended for games, but operates in the launcher. (That being said, unlike the in-game interface, the launcher hasn't had extensive accessibility testing.)  

While there isn't a command line way to create a game (I'd have to think if it's possible with some of the GUI work Ren'Py does when generating a game), one thing you can do with it is launch the project once it's created: 

./renpy.exe /path/to/project 

or 

./renpy.sh /path/to/project 

Should cause the project to start running, at which point you can hit 'v' again to enable self voicing.

**Comment by PhoenixTilea** [comment]
I had no idea self-voicing also worked in the launcher. Thank you for pointing that out. It would be very nice to be able to rely on the full functionality of my screen reader for RenPy's launcher and games, but as I was able to do everything I needed to in the launcher - setting up preferences and creating a new game - with the arrow and enter keys, I'll consider this a win. It's far better than the many, many, many programs and even websites that are just impossible to use.

Thanks again! I'm so used to just getting no response or empty promises that it's very refreshing to actually be told there's a solution.
